client/packages/system/src/MasterList/ListView/ListView.tsx

```diff
@@ -10,29 +10,39 @@ import {
   createQueryParamsStore,
   useUrlQueryParams,
   TooltipTextCell,
+  useAuthContext,
 } from '@openmsupply-client/common';
 import { Toolbar } from './Toolbar';
 import { AppBarButtons } from './AppBarButtons';
 import { MasterListRowFragment, useMasterLists } from '../api';
 
 const MasterListComponent: FC = () => {
```

Comment:
Just a side quest... but can you remove FC? No longer supported in React 18~

Reply:
Done~ https://github.com/msupply-foundation/open-msupply/pull/7909/commits/f803486bfadf448d87e2c04649a476f32383e2a3

Reply:
Thanks!